lib/brains/forms.py

```diff
@@ -1,5 +1,40 @@
 from django import forms
+from django.core.exceptions import ObjectDoesNotExist
+
+from lib.buyers.models import Buyer
 
 
 class BuyerForm(forms.Form):
     uuid = forms.CharField(max_length=255)
+
+
+class PaymentMethodForm(forms.Form):
+    buyer_uuid = forms.CharField(max_length=255)
+    nonce = forms.CharField(max_length=255)
+
+    def clean_buyer_uuid(self):
+        data = self.cleaned_data
+
+        try:
+            self.buyer = Buyer.objects.get(uuid=data['buyer_uuid'])
+        except ObjectDoesNotExist:
+            raise forms.ValidationError('Buyer does not exist.')
+
+        try:
+            self.braintreebuyer = self.buyer.braintreebuyer
+        except ObjectDoesNotExist:
+            raise forms.ValidationError('Braintree buyer does not exist.')
+
+        return data
+
+    @property
+    def braintree_data(self):
+        return {
+            'customer_id': str(self.braintreebuyer.braintree_id),
+            'payment_method_nonce': self.cleaned_data['nonce'],
+            'token': str(self.buyer.pk),
```

Comment:
this doesn't look right. I think it should be the primary key of the payment method object saved in our database, not the pk of the buyer.